tests/pfcwd/test_pfcwd_warm_reboot.py

```diff
@@ -405,12 +462,21 @@ def pfcwd_wb_helper(self, testcase_actions, setup_pfc_test, fanout_graph_facts,
             for p_idx, port in enumerate(self.ports):
                  logger.info("")
                  logger.info("--- Testing on {} ---".format(port))
-                 self.setup_test_params(port, setup_info['vlan'])
+                 self.setup_test_params(port, setup_info['vlan'], p_idx)
                  for q_idx, queue in enumerate(self.pfc_wd['queue_indices']):
                      if not t_idx or storm_deferred:
                          if not q_idx:
                              self.storm_handle[port] = dict()
-                         self.storm_setup(port, queue, storm_defer=(bitmask & 4))
+                         self.storm_handle[port][queue] = None
+
+                         # setup the defer parameters if the storm is deferred currently
+                         if (bitmask & 4):
```

Comment:
Where does 4 come from? Can we make this an `<logical name>_MASK` constant?

Reply:
refer to line 461. I will come back with another PR to use names instead of numbers